src/app/GitUI/NBugReports/BugReportInvoker.cs

```diff
@@ -243,9 +246,15 @@ private static void ReportFailedToLoadAnAssembly(FileNotFoundException exception
             restartButton.Click += (_, _) => RestartGE();
             page.Buttons.Add(restartButton);
 
-            TaskDialogCommandLinkButton reportButton = new(text: TranslatedStrings.ReportIssue, descriptionText: TranslatedStrings.ReportReproducedIssueDescription);
-            reportButton.Click += (_, _) => ShowNBug(OwnerForm, exception, isExternalOperation: false, isTerminating);
-            page.Buttons.Add(reportButton);
+            // Only show the report button for non-.NET framework assemblies
+            // .NET framework assembly errors are typically transient (caused by Patch Tuesday updates)
+            // and should not generate NBug reports
+            if (!isDotNetFrameworkAssembly)
```

Comment:
There is no need for `bool isDotNetFrameworkAssembly`.
```suggestion
            if (!IsDotNetFrameworkAssembly(fileName))
```

Reply:
@copilot take note